sdb/commands/stacks.py

```diff
@@ -152,7 +152,7 @@ def __init__(self,
         self.match_state = ""
 
     @classmethod
-    def _init_parser(cls, name: str) -> argparse.ArgumentParser:
+    def _init_parser(cls, name: str) -> argparse.ArgumentParser():
```

Comment:
Pressumably `mypy` bothered you about this change. We've found it to be extremely inconsistent with false-positives being introduced at every new release thus we've frozed our version of mypy in our Github Actions to `0.730` - https://github.com/delphix/sdb/blob/master/.github/workflows/main.yml#L100 . You may want to do the same in your python environment for sdb locally.

Reply:
Thanks - it sure did!